tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_s3_backend/py_build.sh

```diff
@@ -20,5 +19,3 @@ pip install -r ${temp_path}/requirements.txt -t ${temp_path}/.
 pushd ${build_path}/tmp_building/ && zip -r $output_name . && popd
 mv "${build_path}/tmp_building/${output_name}" "${build_path}/$output_name"
 rm -rf ${build_path}/tmp_building
-pwd
```

Comment:
maybe we address this elsewhere in a different commit, but this will cause issues on windows?

Reply:
Yup you're right that this was addressed in a different commit: https://github.com/aws/aws-sam-cli/pull/4311#discussion_r998592599

Basically: a different project will be used with Windows